tensorflow/lite/kernels/atan2_test.cc

```diff
@@ -62,13 +69,77 @@ class Atan2Model : public tflite::SingleOpModel {
   int output_;
 };
 
+template <>
+std::vector<Eigen::half> Atan2Model::GetOutput(
+    const std::vector<Eigen::half>& y, const std::vector<Eigen::half>& x) {
+  PopulateTensor<Eigen::half>(y_, y);
+  PopulateTensor<Eigen::half>(x_, x);
+  Invoke();
+  return ExtractVector<Eigen::half>(output_);
+}
+
+template <>
+std::vector<Eigen::bfloat16> Atan2Model::GetOutput(
+    const std::vector<Eigen::bfloat16>& y,
+    const std::vector<Eigen::bfloat16>& x) {
+  PopulateTensor<Eigen::bfloat16>(y_, y);
+  PopulateTensor<Eigen::bfloat16>(x_, x);
+  Invoke();
+  return ExtractVector<Eigen::bfloat16>(output_);
+}
```

Comment:
Why are these specializations necessary? They seem to implement the template exactly as default.

Reply:
Yes those weren't needed and I have made the changes.